src/games/strategy/triplea/ai/proAI/ProAI.java

```diff
@@ -116,14 +124,14 @@ public static Logger getLogger() {
 
   public static void gameOverClearCache() {
     // Are static, clear so that we don't keep the data around after a game is exited
-    ProBattleUtils.clearData();
+    concurrentCalc.setGameData(null);
```

Comment:
Need to set null to clear data when the game is over so the battle calc doesn't keep any references. I don't want to change the IOddsCalculator interface at this point to create any new methods.